### ISSUE TITLE
$this can not be used in static methods

### ISSUE THREAD

**Issue by cdCorey** [k1sul1/woocommerce-readonly-role#4]
https://github.com/k1sul1/woocommerce-readonly-role/blob/6d95ec4b7cd652719709da6bc40e6990c40eb14f/woocommerce-readonly-role.php#L56

**Comment by k1sul1** [comment]
Yep. You're right. Role removing probably  never worked because of this. 

Since this is PHP, I guess the fix is to replace `$this` with `$GLOBALS['woocommerce_readonly_role']`, since the class is always instantiated to a global. 

I don't have anything to test this on right now and I'm terribly busy with work that pays my bills, so feel free to send a PR with the fix I proposed if you happen to test it.
